tests/test_experiments.py

```diff
@@ -1,14 +1,40 @@
 """Smoke tests for bash scripts in experiments/"""
+import os
+import pathlib
 import subprocess
 
 import pytest
 
+SCRIPT_NAMES = (
+    "bc_benchmark.sh",
```

Comment:
I've added `dagger_benchmark.sh` to the list. `rollouts_from_policies.sh` I now see is a bit awkward because you need to set output to some temporary directory, so maybe not worth the added test coverage.

Reply:
I must have forgotten `dagger_benchmark.sh`.

Reply:
I'll add a TODO for `rollouts_from_policies.sh`.

Reply:
I added `rollout_from_policies.sh` in.